commands/load_balancers.go

```diff
@@ -513,3 +538,37 @@ func buildRequestFromArgs(c *CmdConfig, r *godo.LoadBalancerRequest) error {
 
 	return nil
 }
+
+func waitForActiveLoadBalancer(lbs do.LoadBalancersService, lbID string) error {
+	const maxAttempts = 180
+	const wantStatus = "active"
+	attempts := 0
+	printNewLineSet := false
+
+	for i := 0; i < maxAttempts; i++ {
+		if attempts != 0 {
+			fmt.Fprint(os.Stderr, ".")
+			if !printNewLineSet {
+				printNewLineSet = true
+				defer fmt.Fprintln(os.Stderr)
+			}
+		}
+
+		lb, err := lbs.Get(lbID)
+		if err != nil {
+			return err
+		}
+
+		if lb.Status == wantStatus {
+			return nil
+		}
+
```

Comment:
We should also check if the status is `errored`. If so, we can exit early with an error rather than waiting for the timeout.

Reply:
You are right, I will change it.